types/enzyme/index.d.ts

```diff
@@ -597,6 +598,13 @@ export class ReactWrapper<P = {}, S = {}, C = Component> {
      * Returns a wrapper with the direct parent of the node in the current wrapper.
      */
     parent(): ReactWrapper<any, any>;
+
+    /**
+     * If a wrappingComponent was passed in options,
+     * this methods returns a ReactWrapper around the rendered wrappingComponent.
+     * This ReactWrapper can be used to update the wrappingComponent's props and state
+     */
+    getWrappingComponent: () => ReactWrapper;
```

Comment:
I’m supposing this returns `undefined` in some circumstances, but the same can be said about almost literally every other method here—I think conveniently “optimistic” types like these make sense for a testing library. (Just thinking out loud)